makemkv/makemkv.py

```diff
@@ -237,9 +238,15 @@ def _translate_codes(
                 # convert 3-letter language codes to 2-letter codes
                 value = Lang(value).pt1
 
+            if key == "framerate":
+                # convert "##.### (#####/####)" to int string
+                if re.search("^\d+\.\d*\s\(\d+/\d+\)$", value):
+                    value = value.split(" ")[0]
+                value = str(round(float(value)))
+
         if code:
             return_value = SPECIAL_VALUES.get(code, value)
-        elif value.strip('"').isdecimal():
+        elif value.strip('"').isdecimal() and not id in keep_str:
```

Comment:
```suggestion
        elif key == "segments_map":
            return_value = [int(item) for item in value.strip('"').split(",")]
        elif value.strip('"').isdecimal():
```

Reply:
I think it would make sense to replace all those `value.strip('"')` with a single `value = value.strip('"')` above `if code:`.

Reply:
> I think it would make sense to replace all those `value.strip('"')` with a single `value = value.strip('"')` above `if code:`.

Nevermind, the `"` chars are already stripped by the regex that splits each line into multiple values.